configs/test/bigquery/datasets.yaml

```diff
@@ -225,3 +225,29 @@ resources:
             type: FLOAT
           - name: engine
             type: STRING
+  - name: cross-pollination-statistics-table
+    type: gcp-types/bigquery-v2:tables
+    properties:
+      datasetId: $(ref.main-dataset.datasetReference.datasetId)
+      tableReference:
+        tableId: cross-pollination-statistics
+      timePartitioning:
+        type: DAY
+      schema:
+        fields:
+        - name: project_qualified_name
```

Comment:
how do we track what corpus was used for pollination? this field is the name of the target which we attempted to add more corpus to, i.e. it's "destination". I think tracking "sources" is also important. For example, after recording this for a while, we could try auto-deriving the tags based on what fuzz targets have helped each other through global cross polination.

Reply:
The problem with this is that multiple sources are used for pollination not just one. Do we want a field in the table that is a list of all soruce corpora?

Reply:
Yes, I think it's fine to have a string where we'll have multiple names listed. There is substring functionality in BQ language, should be fine if we need to query those.

Reply:
Added. Right now the logic isn't there to populate the tags field. That will come in the next CL.